### PR TITLE
issue-109

### LINKED ISSUE #109: Disable individual Fault Tolerance annotation using external config
Based on the current state of Fault Tolerance configuration, It is not possible to disable specific annotation individually or globally. 

Possible solution could be :

1. Introduce an enabled element with default value true in annotation to disable it using external config source.
e.g : `com.acme.test.MyClient/serviceA/CircuitBreaker/enabled=false`
2. The annotation can be disabled via system properties in the naming convention of `<classname>/<methodname>/<annotation>=false`
e.g : `com.acme.test.MyClient/serviceA/CircuitBreaker=false`

### PR DESCRIPTION
**Pull request by antoinesd**
Fix #109 by Adding spec and test

### CODE REVIEW

File: spec/src/main/asciidoc/configuration.asciidoc [hunk 1]
```diff
@@ -107,6 +107,48 @@ public class MyClient{
...
+
+Policy will be disabled even if the policy is also defined at class level
+
+* Disabling a policy at class level
+
+A policy can be disabled at class level with the following config property and value:
+
+`<classname>/<annotation>/enabled=false`
+
+For instance the following config will disable fallback policy on `com.acme.test.MyClient` class:
```
Comment (Emily-Jiang) [inline spec/src/main/asciidoc/configuration.asciidoc]: Fallback is only a method level annotation. You might choose Retry for this explanation.

File: tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/disableEnv/DisableAnnotationGloballyTest.java [hunk 2]
```diff
@@ -0,0 +1,141 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2016-2017 Contributors to the Eclipse Foundation
```
Comment (Emily-Jiang) [inline tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/disableEnv/DisableAnnotationGloballyTest.java]: The copyright year needs to be 2016-2018

**Comment by antoinesd** [comment]
Added mention of config pattern without `enabled=false` being non-portable

**Comment by antoinesd** [comment]
@Emily-Jiang corrected property names and added test for asynchronous policy deactivation

**Pull request merged**